Gopkg.toml

```diff
@@ -57,7 +57,7 @@
   name = "github.com/giantswarm/microstorage"
 
 [[constraint]]
-  branch = "informer"
+  branch = "master"
```

Comment:
No idea why this was pinned differently. Here we go back to master and apply the necessary structural changes.